### ISSUE TITLE
Preprocessing BP4D dataset

### ISSUE THREAD

**Issue by Magicyu-2015** [ZhiwenShao/ADLD#1]
Dear author, Thank you for providing code to the community! I have a question for the preprocessing BP4D dataset.  We follow your instructions to preprocessing the EmotioNet database. How do we preprocessing the BP4D database? Also using python dataset/face_transform.py? Can you provide details for BP4D database? Looking for your reply! Thanks!

**Comment by ZhiwenShao** [comment]
The preprocessing of EmotioNet and BP4D are same, both using the dataset/face_transform.py.

**Comment by Magicyu-2015** [comment]
Thank you for your reply! It does mean I need to generate the 49 facial landmark location file from the MAT file in BP4D dataset. Is it right? By the way, I would like to know how do you generate the BP4D_train_path.txt. The order in this file is totally different from the MAT file in BP4D dataset.

**Comment by ZhiwenShao** [comment]
49 facial landmark locations should be in the format: x1 y1 x2 y2... in a line for each image. The BP4D_train_path is shuffled, and shuffle or not does not matter because we set shuffle=True in the train dataloader. The BP4D_train_path contains 28 subjects, and you just need to select these subjects from the BP4D dataset to conduct processing. For the convenience, I have updated the BP4D_train_path with ordered image list.

**Comment by huqiaoping** [comment]
I found BP4D does not provide the landmarks for some images, e.g. M010/T6/958.jpg.  How did you get landmarks of these images? @ZhiwenShao

**Comment by ZhiwenShao** [comment]
We just remove these images from the training set.

**Comment by linger-8023** [comment]
Dear auther, I'm finding  the BP4D dataset but don't  find it. Could you share?
